config.yaml

```diff
@@ -38,198 +38,106 @@ docusaurus:
-              name: content/redirections
-          - Pipes:
-              location: content/chapters/io/lab
-              name: content/pipes
-          - Local I/O in Action:
-              location: content/chapters/io/lab
-              name: content/local-io-in-action
-          - Remote I/O:
-              location: content/chapters/io/lab
-              name: content/remote-io
-          - Networking 101:
-              location: content/chapters/io/lab
-              name: content/networking-101
-          - Client-Server Model:
-              location: content/chapters/io/lab
-              name: content/client-server-model
-          - Beyond Network Sockets:
-              location: content/chapters/io/lab
-              name: content/beyond-network-sockets
-          - File Mappings:
-              location: content/chapters/io/lab
-              name: content/file-mappings
-          - I/O Internals:
-              location: content/chapters/io/lab
-              name: content/io-internals
-          - Zero-Copy:
-              location: content/chapters/io/lab
-              name: content/zero-copy
-          - Asynchronous I/O:
-              location: content/chapters/io/lab
-              name: content/async-io
-          - I/O Multiplexing:
-              location: content/chapters/io/lab
-              name: content/io-multiplexing
-          - Arena:
-              location: content/chapters/io/lab
-              name: content/arena
+            path: content/chapters/compute/lab/content
+            extra:
+              - ../media
+              - ../quiz
+            subsections:
+              - Overview: overview.md
+              - Hardware Perspective: hardware-perspective.md
+              - Processes: processes.md
+              - Threads: threads.md
+              - Processes-threads-apache2: processes-threads-apache2.md
+              - Copy-on-Write: copy-on-write.md
+              - Synchronization: synchronization.md
+              - Scheduling: scheduling.md
+              - Arena: arena.md
+        - I%2FO:
+            path: content/chapters/io/lab/content
+            extra:
+              - ../media
+              - ../quiz
+            subsections:
+              - Overview: overview.md
+              - File Handlers: file-handlers.md
+              - File Descriptors: file-descriptors.md
+              - Redirections: redirections.md
+              - Pipes: pipes.md
+              - Local I%2FO in Action: local-io-in-action.md
+              - Remote I%2FO: remote-io.md
+              - Networking 101: networking-101.md
+              - Client-Server Model: client-server-model.md
+              - Beyond Network Sockets: beyond-network-sockets.md
+              - File Mappings: file-mappings.md
+              - I%2FO Internals: io-internals.md
+              - Zero-Copy: zero-copy.md
+              - Asynchronous I%2FO: async-io.md
+              - I%2FO Multiplexing: io-multiplexing.md
```

Comment:
It's rather unfortunate that you have to URL-encode "I/O". Maybe it's better to just call it "IO".

Reply:
I know, but in order to simplify the config, I had to give `/` the "meaning" of subfolder (see [here](https://open-education-hub.github.io/openedu-builder/plugins/docusaurus#structure), at the end). URL-encoding was really handy at the time, but I'll think of a way to make it more human-readable in a future version. Should I leave it like this or change it to "IO"?

Reply:
Chnage it to IO. Otherwise it looks weird and forced.

Reply:
Done